src/Analysis/Codelyzer.Analysis.Model/AccessorBlock.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Newtonsoft.Json;
```

Comment:
maybe we can put all these new vb specific models into a VisualBasic folder to keep things more organized?